src/screens/FeedScreen/Header.js

```diff
@@ -135,7 +135,11 @@ const _renderAnonimity = ({
               </View>
             ) : null}
             <View style={[styles.imageAnonymContainer]}>
-              <AnonymousAvatar anonUserInfo={anonUserInfo} version={version} />
+              <AnonymousAvatar
+                karmaScore={karmaScore}
+                anonUserInfo={anonUserInfo}
+                version={version}
+              />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `karmaScore` prop has been added to the `AnonymousAvatar` component. Ensure that this prop is properly validated and handled within the component itself. If `karmaScore` can be `null` or `undefined`, make sure to handle these cases appropriately in the `AnonymousAvatar` component to prevent potential crashes or unexpected behavior.
```diff
<AnonymousAvatar
+  karmaScore={karmaScore}
  anonUserInfo={anonUserInfo}
  version={version}
/>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->